lib/bookingsync_portal/version.rb

```diff
@@ -1,3 +1,3 @@
 module BookingsyncPortal
-  VERSION = "1.0.1"
+  VERSION = "2.0.1"
```

Comment:
why 2.0.1? this is not a breaking change?

Reply:
@Azdaroth ,

Well, as you may see on changelog, we agreed to bump gem ver to 2.0.0 on breaking change of dependencies.

But seems version was not bumped here at the end.

To keep versioning in right way I have to increase the latest gem version annotated in changelog, i.e. 2.0.0 -> 2.0.1

Makes sense?

Reply:
Oh, wow... indeed, definitely makes sense then 👍